code/modules/client/preference_setup/preview.dm

```diff
@@ -0,0 +1,161 @@
+#define MODIFICATION_ORGANIC 1
+#define MODIFICATION_SILICON 2
+#define MODIFICATION_REMOVED 3
+
+datum/preferences
+	var/req_update_icon = 1
+	var/icon/preview_icon  = null
+	var/icon/preview_south = null
+	var/icon/preview_north = null
+	var/icon/preview_east  = null
+	var/icon/preview_west  = null
+	var/preview_dir = SOUTH
+
+	proc/update_preview_icon()
```

Comment:
Proc defines should contain full type path, see [contribution guide.](https://github.com/Endless-Horizon/CEV-Eris/blob/master/CONTRIBUTING.md)